Tests/GettableSpec.swift

```diff
@@ -29,5 +52,43 @@ class GettableSpec: QuickSpec {
             
             itBehavesLike("a Restable") { context }
         }
+        
+        describe("A gettable with a dateDecodingStrategy") {            
+            it("It decodes the response data according to the dateDecodingStrategy") {
+                let gettable = DateRequest()
+                let currentDateString = "2018-06-12T20:00:00.000-04:00"
+                let currentDate = gettable.dateFormatter().date(from: currentDateString)!
+                
+                do {
+                    // encode the date
+                    let encoder = JSONEncoder()
+                    encoder.dateEncodingStrategy = .formatted(gettable.dateFormatter())
+                    let data = try encoder.encode(["date": currentDateString])
+                    var dateResponse: DateResponse? = nil
```

Comment:
To limit the amount of work we're doing in this test, and in an attempt to avoid failure points and not conflate what we're testing (i.e. keep it focused), I think it would be best to _not_ generate the JSON data using the other parts of the system. As such, you should be able to simply do the following:

```Swift
let data = """
{ "date": "2018-06-12T20:00:00.000-04:00" }
""".data(using: .utf8)!
```

Reply:
👍